prisma/schema.prisma

```diff
@@ -99,31 +99,32 @@ model CollectionAuthor {
 }
 
 model Label {
-  id          Int       @id @default(autoincrement())
-  externalId  String    @default(uuid()) @db.VarChar(255)
-  name        String    @unique @db.VarChar(255)
-  createdAt   DateTime  @default(now()) @db.DateTime(0)
-  createdBy   String    @db.VarChar(255)
-  updatedAt   DateTime  @updatedAt
-  updatedBy   String?   @db.VarChar(255)
+  id          Int               @id @default(autoincrement())
+  externalId  String            @default(uuid()) @db.VarChar(255)
+  name        String            @unique @db.VarChar(255)
+  createdAt   DateTime          @default(now()) @db.DateTime(0)
+  createdBy   String            @db.VarChar(255)
+  updatedAt   DateTime          @updatedAt
+  updatedBy   String?           @db.VarChar(255)
+  collections CollectionLabel[]
+
 
   @@unique([externalId])
   @@index([id])
```

Comment:
Since this is a refactor PR already, it would be good to remove this - we don't need to set up an index on a primary key field, it comes for free.